### ISSUE TITLE
UI glitches

### ISSUE THREAD

**Issue by czytelny** [gilmarsquinelato/i18n-manager#148]
Hi, first of all great idea, thank you for creating this app! 
I have small problem UI, though. I downloaded app v1.8.2 from "releases" for MacOS.
My translation files looks like this: 

```
{
  "board": {
    "no_items_yet": "No items on this board yet",
    "empty_board": "This board is empty"
  }
}
```

Seems like a problem with icons ? 
<img width="1301" alt="Screenshot 2019-06-22 at 10 17 29" src="https://user-images.githubusercontent.com/5399256/59961396-8282cb00-94d7-11e9-80e9-1d36d805c074.png">

Regards

/// edit
After some time I run the same app again and now it works fine.. icons are in place :) But it would be nice if it works from first run.

**Comment by gilmarsquinelato** [comment]
Hi @czytelny, thanks!
Yeah, I saw this occurring some times, unfortunately.
At this time I'm developing the v2 of this app, and it will be changed from Angular to React performance improvements and the change from Material Design to Ant Design.
Mainly the last one I believe that the UI issues will be solved, due to the entire stylesheet will be local.

So I would like a little bit of patience to get the new version of that app 😄

**Comment by czytelny** [comment]
Sounds great. Ant design is awesome. And with React onboard I'd love to contribute :) looking forward for v2 then :) cheers

**Comment by gilmarsquinelato** [comment]
Reopening for tracking purposes and notify when the V2 come.

**Comment by gilmarsquinelato** [comment]
Hi @czytelny, just released the v2.0, can you test it and check if everything goes ok?
If it's ok, we can close this issue :)

**Comment by czytelny** [comment]
Sorry, completely forgot to answer. Looks good now, definitely will use ! 👍
